### PR TITLE
[VTAdmin] `Validate`, `ValidateShard`, `ValidateVersionShard`, `GetFullStatus`

### PR DESCRIPTION
**Pull request by notfelineit**
## Description
This PR completes "VTAdmin Parity" by implementing:
- `Validate`
- `ValidateShard`
- `ValidateVersions`
- `GetFullStatus`

**Validate**
Validate is added to the clusters list, since validation occurs at the cluster level:
validate

**ValidateShard**
validateShard

**GetFullStatus**
`GetFullStatus` replaces `Replication Status` since it also includes that in its payload:
Screen Shot 2022-10-04 at 1 47 28 PM

**ValidateVersionShard**
validateVersionShard

## Related Issue(s)
- [x] https://github.com/vitessio/vitess/issues/8741
- [x] https://github.com/vitessio/vitess/issues/11258
- [x] https://github.com/vitessio/vitess/issues/8718
- [x] https://github.com/vitessio/vitess/issues/8718

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported **Should be backported to 15**
-   [x] Tests were added or are not required **Not necessary ATM**
-   [x] Documentation was added or is not required **Not required**

### CODE REVIEW

**Comment by deepthi** [comment]
LGTM visually. Let's get a code review as well.

File: web/vtadmin/src/components/dialog/Dialog.tsx [hunk 1]
```diff
@@ -121,7 +121,6 @@ const Dialog: React.FC<DialogProps> = ({
                                             className="btn"
                                             onClick={() => {
                                                 onConfirm && onConfirm();
```
Comment (no-itsbackpack) [inline web/vtadmin/src/components/dialog/Dialog.tsx:123]: we could use the new es syntax
```suggestion
                                                onConfirm?.();
```

**Review by no-itsbackpack — APPROVED** [review]
✅ from #surfaces on UI changes 🚀 🚀 🚀

File: web/vtadmin/src/components/dialog/Dialog.tsx (hunk 1 above)
Comment (notfelineit) [reply web/vtadmin/src/components/dialog/Dialog.tsx]: 👍

**Review by ajm188 — COMMENTED** [review]
couple nits but also some required minor changes on the golang side, but looking great!

File: go/vt/vtadmin/api.go [hunk 2]
```diff
@@ -771,6 +775,25 @@ func (api *API) GetClusters(ctx context.Context, req *vtadminpb.GetClustersReque
...
+func (api *API) GetFullStatus(ctx context.Context, req *vtadminpb.GetFullStatusRequest) (*vtctldatapb.GetFullStatusResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "API.GetFullStatus")
+	defer span.Finish()
+
+	c, err := api.getClusterForRequest(req.ClusterId)
+	if err != nil {
+		return nil, err
+	}
+
+	if !api.authz.IsAuthorized(ctx, c.ID, rbac.TabletResource, rbac.GetAction) {
```
[the remark this answers is not in the document]
Comment (notfelineit) [reply go/vt/vtadmin/api.go]: Added a `TabletFullStatusResource`. It can be changed if another resource seems better.

**Review by ajm188 — APPROVED** [review]

**Pull request merged**